wp-parsely.php

```diff
@@ -1649,29 +1648,19 @@ private function get_current_url( $post = 'nonpost', $post_id = 0 ) {
 			 * @param int    $post_id   ID of the post you want to get the URL for. May be 0, so $permalink will be
 			 *                          for the global $post.
 			 */
-			$permalink        = apply_filters( 'wp_parsely_permalink', $permalink, $post, $post_id );
-			$parsed_canonical = wp_parse_url( $permalink );
-			// handle issue if wp_parse_url doesn't return good host & path data, fallback to page url as a last resort.
-			if ( isset( $parsed_canonical['host'], $parsed_canonical['path'] ) ) {
-				$canonical = $scheme . $parsed_canonical['host'] . $parsed_canonical['path'];
-			} elseif ( isset( $_SERVER['HTTP_HOST'], $_SERVER['REQUEST_URI'] ) ) { // Input var okay.
-				$canonical = $scheme . sanitize_text_field( wp_unslash( $_SERVER['HTTP_HOST'] ) ) . sanitize_text_field( wp_unslash( $_SERVER['REQUEST_URI'] ) ); // Input var okay.
-			}
+			$url = apply_filters( 'wp_parsely_permalink', $permalink, $post, $post_id );
```

Comment:
Can this filter be made available to the other branch as well? Does it make to do that?

Reply:
I keep it as-is like what the original code is doing.
I've looked at this filter again. It is passing `$post_id`, which is not available in another conditional branch so we should not make it available outside of the current scope.